pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/InnvilgelseAvAlderspensjon.kt

```diff
@@ -0,0 +1,580 @@
+                        )
+                    }
+                }
+
+                showIf(saksbehandlerValg.egenOpptjening and not(gjenlevenderettAnvendt)) {
+                    // beregningAPGjRettOpptjEgen_002
+                    title1 {
+                        text(
+                            Bokmal to "Gjenlevenderett i alderspensjon",
+                            Nynorsk to "Attlevenderett i alderspensjon",
+                            English to "Survivor's rights in retirement pension"
+                        )
+                    }
+                    paragraph {
+                        textExpr(
+                            Bokmal to "I beregningen vår har vi tatt utgangspunkt i din egen opptjening. Dette gir deg en høyere pensjon enn om vi hadde tatt utgangspunkt i pensjonsrettighetene du har etter ".expr()
+                                    + avdodNavn + ".",
+                            Nynorsk to "I vår berekning har vi teke utgangspunkt i di eiga opptening. Dette gir deg ein høgare pensjon enn om vi hadde teke utgangspunkt i pensjonsrettane du har etter ".expr()
+                                    + avdodNavn + ".",
+                            English to "We have based our calculation on your own earnings. This gives you a higher pension than if we had based it on the pension rights you have after ".expr()
+                                    + avdodNavn + "."
+                        )
+                    }
+                }
+            }
+
+            showIf(privatAFPErBrukt) {
+                includePhrase(
+                    AfpPrivatErBrukt(uttaksgrad = uttaksgrad)
+                )
+            }
+
+            showIf(afpPrivatResultatFellesKontoret) { includePhrase(SoktAFPPrivatInfo) }
+
+            showIf(erEksportberegnet and not(eksportForbud) and not(minst20ArTrygdetid)) {
+                // innvilgelseAPUnder20aar
+                paragraph {
+                    textExpr(
+                        Bokmal to "Du har ikke vært medlem i folketrygden i minst 20 år. Da har du ikke rett til å få utbetalt hele alderspensjonen din når du bor i ".expr()
+                                + fakstiskBostedsland + ".",
+                        Nynorsk to "Du har ikkje vore medlem i folketrygda i minst 20 år. Da har du ikkje rett til å få utbetalt heile alderspensjonen din når du bur i ".expr()
+                                + fakstiskBostedsland + ".",
+                        English to "You have not been a member of the National Insurance Scheme for at least 20 years. You are therefore not eligible for a full retirement pension while living in ".expr()
+                                + fakstiskBostedsland + "."
+                    )
+                }
+            }
+
+            showIf(
+                erEksportberegnet and not(eksportForbud) and harAvdod and not(minst20ArTrygdetidKap20) and not(
```

Comment:
Litt anna showIf her også enn i doksys, med vilje?

Reply:
Ja, endret visningslogikk med vilje. Eksport boolean kan ikke være tom i brevbaker, så settes "false", og jeg vil ha en ekstra skjekk for at det finnes en avdod - harAvdod.